<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InscriptionWindow</class>
 <widget class="QDialog" name="InscriptionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Location</string>
   </property>
   <property name="buddy">
    <cstring>editLocation</cstring>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="dialog">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>300</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="editLocation">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="editMail">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Prenom</string>
   </property>
   <property name="buddy">
    <cstring>editFirstName</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="buddy">
    <cstring>editLogin</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="editUrl">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="editFirstName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Url</string>
   </property>
   <property name="buddy">
    <cstring>editUrl</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
   <property name="buddy">
    <cstring>editMail</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="editLogin">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="editName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>SERVICE DES INSCRIPTIONS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom</string>
   </property>
   <property name="buddy">
    <cstring>editCompany</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Compagnie</string>
   </property>
   <property name="buddy">
    <cstring>editName</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="editCompany">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>editCompany</tabstop>
  <tabstop>editFirstName</tabstop>
  <tabstop>editLogin</tabstop>
  <tabstop>editLocation</tabstop>
  <tabstop>editUrl</tabstop>
  <tabstop>editMail</tabstop>
  <tabstop>editName</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
